src/api/app/views/webui2/webui/user/_involvement.html.haml

```diff
@@ -66,7 +66,8 @@
               - owned.each do |package_name, project_name|
                 %tr
                   %td
-                    = link_to(package_name, package_show_path(package: package_name, project: project_name))
+                    - if package_name
+                      = link_to(package_name, package_show_path(package: package_name, project: project_name))
```

Comment:
https://api.rubyonrails.org/v5.2.1/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to_if anyone? :)

Reply:
Good idea:thumbsup:

Reply:
Either I am doing something wrong or `link_to_if` doesn't like path helpers. At least not if the condition is falsy.

After checking the [code](https://github.com/rails/rails/blob/fc5dd0b85189811062c85520fd70de8389b55aeb/actionview/lib/action_view/helpers/url_helper.rb#L439) I also noticed that this is not exactly the same. With `link_to_if` we would still render the `package_name` variable.

I am a bit in a hurry right now. I'll double check later on.

Reply:
ah, I see. Sorry for the wrong lead